va-gov/pages/education/verify-school-enrollment.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Verify School Enrollment
+order: 7
+spoke: Manage Benefits
+href: https://www.gibill.va.gov/wave/index.do
+private: true
+target: _blank
+---
```

Comment:
The spreadsheet says `Verify School Enrollment`, but the ticket says `Verify School Attendance`. Not sure which is right on that one. 

Ticket - https://app.zenhub.com/workspace/o/department-of-veterans-affairs/vets.gov-team/issues/14088

Reply:
spoke to mikki on slack @ncksllvn it is `Verify School Enrollment`